source/documentation/orgs_spaces_users/index.md

```diff
@@ -0,0 +1,215 @@
+
+## Spaces
+
+An org is divided into one or more spaces. A space is a shared location for app development, deployment and maintenance. Apps and services are pushed to spaces and hosted within them.
+
+For example, you might have spaces for development and production versions of your app. When we set up your org, we create a default sandbox space you can use for experimenting with the PaaS.
+
+Run `cf spaces` to see the spaces you can access in your current org.
+
+[Org managers](/#org-managers) can create new spaces within an org. Run `cf org-users ORGNAME` to find out who has that role, where ORGNAME is the name of the org. You will see a list of users and their roles. You must be a member of that org to see the list of users.
+
+## Users and user roles
+
+Users are members of your team who use the command-line tool to manage apps on the GOV.UK PaaS.
+
+Users are assigned roles which have different permissions for accessing and managing orgs and spaces. A user can have:
+
+- one or multiple roles
+- more than one role within the same org or space
+- different roles in different orgs and spaces
+
+Run `cf org-users ORGNAME` to find out details of the users in your org, where `ORGNAME` is the name of the org.
+
+Run `cf orgs` to see which orgs you can access.
+
+To change which user accounts have access, your user account must have the Org Manager role.
+
+Although there are multiple roles, the five most common roles are:
+
+- Org manager
+- Billing manager
+- Space developer
+- Space manager
+- Space auditor
+
+Refer to the [Cloud Foundry documentation on roles and permissions](https://docs.cloudfoundry.org/concepts/roles.html#roles) for more information on these roles and others.
+
+### Org manager
+
+This role applies within an org.
+
+Every org must have at least one org manager. Org managers are primary contacts for the org.
+
+Org managers manage spaces, users and user roles, and approve org changes such as requesting new quotas and enabling paid services.
+
+We create at least one user account with the org manager role as part of the onboarding process. We recommend you have at least two org managers in case one is unavailable.
+
+If you need the org manager role added to a user account, contact us at [gov-uk-paas-support@digital.cabinet-office.gov.uk](mailto:gov-uk-paas-support@digital.cabinet-office.gov.uk).
+
+### Billing manager
+
+This role applies within an org.
+
+Billing managers create and manage billing account and payment information. They can also view users and roles.
+
+You should assign a billing manager to your org before your service moves to production. We will send all payment requests to the billing manager. The org manager can also be the billing manager.
+
+Contact us at [gov-uk-paas-support@digital.cabinet-office.gov.uk](mailto:gov-uk-paas-support@digital.cabinet-office.gov.uk) if you have any questions.
```

Comment:
Jess wanted a call to action here to contact the team, because although billing managers are not mandatory, we want to encourage people to assign billing managers, so having a call to action might mean that we can convince more people to do so.